.github/workflows/test.yaml

```diff
@@ -1,11 +1,13 @@
 name: test
-on: [pull_request, push]
+on: [pull_request]
 jobs:
   run_tests:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
     container: golang:1.16
     steps:
-      - name: Checkout
+      -
+        name: Checkout
         uses: actions/checkout@v2
-      - name: Run tests
+      -
+        name: Run tests
         run: make test
```

Comment:
These changes are unrelated to the release process, but it fixes a bug where the tests were being run twice, triggered by both the `pull_request` event and by the `push` event. GitHub actions doesn't seem to de-duplicate the events.